CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 * Adds support for one-call-buys on shipments and orders via adding the `service` key to both objects
 * Removes the unused `orderBy` parameter from the `Batch` object
-
+* Update the DefaultApiBase to include `v2` at the end and remove `v2` from every request url string
```

Comment:
Nit: This reads a bit nicer. Also, please add a newline after the bullet point so we properly separate each section (the bullet point should have a newline before the next header section)

```suggestion
* Update the `DefaultApiBase` to include `v2` and remove `v2` from every request url string
```